client/test-data/test.locations.mediterranean.locolight.json

```diff
@@ -0,0 +1,55 @@
+{
+  "type": "FeatureCollection",
+  "community": {
+    "meta": {
+      "name": "Mediterranea",
+      "symbol": "MTA",
+      "assets": "QmP2fzfikh7VqTu8pvzd2G2vAd4eK7EaazXTEgqGN6AWoD",
+      "announcementSigner": {"bip340": "0xceda72c19f11aebecf8fb4a0ab3fb2a03e03f283225655baab4728303a560a13"},
+      "rules": "loCoLight"
```

Comment:
I don't like camel case here as it is not intuitive

Reply:
This is substrate standard, we should not go against it: https://github.com/encointer/pallets/blob/1933ce2cc9b196bf8aca13e11c22d8ab254b8a7a/primitives/src/communities.rs#L68

If we had better set up the de-/serialization for the other fields we could simply do: `CommunityMetdata::from_str(&spec["community"]["meta"])`, but we will never be able to do so if we go against the standard here.

Reply:
Ah sorry, I see now that you don't mean the field name, but the enum variant `loCoLight`. This is also the substrate standard, but I see now that serde can handle both, I change it to UpperCamelCase